src/components/ConfigureEmail.tsx

```diff
@@ -1,49 +1,37 @@
 import React, { useEffect, useState } from 'react'
```

Comment:
Ellis wrote a ConfigureEmail component, I just needed to restyle it (and decided to use some available hooks).